lib/util/http/index.ts

```diff
@@ -66,6 +67,14 @@ export class Http {
         },
       ],
     };
+    if (global.appMode) {
```

Comment:
I think we should always set the user agent

Reply:
Looks like we do, and we'd overwrite the one here: https://github.com/renovatebot/renovate/blob/master/lib/util/got/renovate-agent.ts

Reply:
I think this whole section can be removed